ccd-case-generator/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
```

Comment:
We're maintaining the CCD def as a separate module? If so I think it should be ccd-definition or something.

Reply:
I went with a sub module as the ccd-config-generator plugin only works with master and this seemed to be the easiest way to exclude any CcdConfig classes.  We could add some configuration to the plaugin to allow a different source directory if we want to simplify things